sdk/storage_blobs/src/blob/operations/get_blob.rs

```diff
@@ -173,6 +129,62 @@ impl Continuable for GetBlobResponse {
     }
 }
 
+// caclculate the first Range for use at the beginning of the Pageable.
```

Comment:
FYI: even though these are internal APIs they should still be `///` doc comments so that they show up in rust-analyzer.